File: app/src/main/java/no/nav/sosialhjelp/soknad/business/pdfmedpdfbox/PdfGenereringException.java
package no.nav.sosialhjelp.soknad.business.pdfmedpdfbox;

import no.nav.sosialhjelp.soknad.domain.model.exception.SosialhjelpSoknadApiException;

public class PdfGenereringException extends SosialhjelpSoknadApiException {

    public PdfGenereringException(String melding, Throwable e) {
        super(melding, e);
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/pdf/PdfGenereringException.kt
package no.nav.sosialhjelp.soknad.pdf

import no.nav.sosialhjelp.soknad.app.exceptions.SosialhjelpSoknadApiException

class PdfGenereringException(melding: String?, e: Throwable?) : SosialhjelpSoknadApiException(melding, e)
